template-tarot.js

```diff
@@ -70,9 +97,9 @@ var TarotCard = (function(){ //<-- Superclass is parameter, but equals Card
 
 	// Override some other superclass methods and resources:
 
-	var suitNames = ["Cups","Pentacles","Swords","Wands"];
+	var suitNames = ['',"Cups","Pentacles","Swords","Wands"];
 	var rankNames = Card.rankNames(); // subclass ranks are derived from superclass Card,
-	rankNames.splice(10,1,"Page","Knight"); // but Jack gets replaced w. Page+Knight
+	rankNames.splice('',10,1,"Page","Knight"); // but Jack gets replaced w. Page+Knight
```

Comment:
For the rankNames, you might look into JavaScript's `splice` method to see what arguments it's expecting. Don't forget that you can always use `console.log` to see, for example, what the values of `rankNames` are.